UPGRADES.md

```diff
@@ -0,0 +1,94 @@
+- For all other applications, recompile using a supported version of MATLAB Runtime. Refer to the [MATLAB Compiler SDK documentation](https://www.mathworks.com/help/compiler_sdk/mps_dev_test/create-a-deployable-archive-for-matlab-production-server.html) for information on how to create deployable archives for MATLAB Production Server.
+
+See the [MATLAB Production Server on AWS README](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) for a complete list of supported MATLAB Runtime versions for your release. 
+
+### Server Downtime
+These measures can help reduce the time your MATLAB Production Server and deployed applications are inaccessible during the upgrade process: 
+
+- **Use a new network license manager for your new deployment.** This allows you to keep both deployments functional while deployed applications and client applications are migrated. Note that running two instances of MATLAB Production Server simultaneously does require two separate license instances. 
+- **Set a custom URL for your MATLAB Execution Endpoint.** This allows you to upgrade your MATLAB Production Server deployment without needing to update your client applications. Refer to the [“URL Compatibility”](#url-compatibility) section above for instructions on setting a custom DNS name for this endpoint. 
+- **Compile deployed applications using the newest supported MATLAB Runtime version**. MATLAB Production Server supports the six most recent versions of MATLAB Runtime, so applications compiled for existing deployments of recent releases can be immediately migrated to a new deployment without needing to be recompiled. Deployed applications can be updated and redeployed after the upgrade process is complete. 
+
+
+## Upgrade Steps 
+
+### Step 1. Deploy Newest MATLAB Production Server Release from GitHub CloudFormation Template
+When you upgrade the deployment, you can also deploy a new virtual public cloud (VPC) and a new network license manager. This table summarizes the standard workflows.  
+|  | Existing Network License Manager | New Network License Manager |
+| --- | ------------ | ------- |
+| **Existing VPC** | Deploy the “Existing VPC” template, selecting **No** for **DeployLicenseServer**. | Deploy the “Existing VPC” template, selecting **Yes** for **DeployLicenseServer**. |
+| **New VPC** | Not recommended | Deploy the "New VPC" template. |
+
+
+*If using a new VPC*: 
+1. In the GitHub repository for [MATLAB Production Server on AWS](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws), navigate to the [MATLAB release](https://github.com/mathworks-ref-arch/matlab-production-server-on-aws#deploy-reference-architecture-for-your-release) you are deploying. Under “Step 1. Launch Template,” click **Launch Stack**. (For R2023a and later, click the button under New VPC.)
```

Comment:
For consistency between the first bullets in new VPC and existing VPC, we can use either "In the GitHub repository..." or "From the GitHub repository..." in both sentences.